overlord/snapstate/backend_test.go

```diff
@@ -121,7 +121,7 @@ func (f *fakeSnappyBackend) CopySnapData(instSnapPath string, flags snappy.Insta
 
 func (f *fakeSnappyBackend) GenerateSecurityProfile(instSnapPath string) error {
 	f.ops = append(f.ops, fakeOp{
-		op:   "generate-security-profile",
+		op:   "setup-snap-security",
```

Comment:
Can we please do a pass over the whole code to unify the terminology around the new "link-snap" and "setup-snap-security" names, and getting rid of the old ones (generate security profiles, generate wrappers, etc)?